docs/manual/nullness-checker.tex

```diff
@@ -2213,31 +2216,6 @@
 \end{sloppypar}
 
 
-\subsectionAndLabel{Suppressing warnings}{initialization-checking-suppressing-warnings}
```

Comment:
I would not remove this subsection.
Instead, please describe the current system about `nullness` and `initialization` suppression keys, how we'll make this stricter in the future, and how the `-AassumeInitialized` option works.